### ISSUE TITLE
Change mentions of "master" to "main" in Contributing.rst

### ISSUE THREAD

**Issue by BenMcLean981** [pallets-eco/flask-sqlalchemy#1042]
currently, the contributors guide still mentions the origin/master branch.

```
If you're submitting a feature addition or change, branch off of the
    "main" branch.

    .. code-block:: text

        $ git fetch origin
        $ git checkout -b your-branch-name origin/main
```

I am submitting a PR to fix this, and the contributor's guide indicates I should
have an open issue to reference. This is my first time contributing to open-source
 software, so if I've made any mistakes, I would appreciate the feedback!

**Comment by ThiefMaster** [comment]
> I am submitting a PR to fix this, and the contributor's guide indicates I should have an open issue to reference.

This only applies to more significant changes where we don't want people to start development just to find out it's not something we'd like to have. For simple typo-like fixes there's no need to open an issue.

You're not the first one who's doing this though, so maybe we should make it clearer...

**Comment by davidism** [comment]
Fixed in #1087
